src/libtorch.cc

```diff
@@ -947,8 +1021,8 @@ ModelInstanceState::SetInputTensors(
     // The input must be in contiguous CPU/GPU memory.
     std::vector<std::pair<TRITONSERVER_MemoryType, int64_t>> alloc_perference;
     if (device_.is_cpu()) {
-      alloc_perference = {{TRITONSERVER_MEMORY_CPU_PINNED, 0},
-                          {TRITONSERVER_MEMORY_CPU, 0}};
+      alloc_perference = {
```

Comment:
Can you run the format.py from the common repo? More instructions here:
https://github.com/triton-inference-server/server/blob/main/CONTRIBUTING.md#coding-convention